packages/data-stores/src/launchpad-navigator/actions.ts

```diff
@@ -0,0 +1,49 @@
+import apiFetch from '@wordpress/api-fetch';
+import { select } from '@wordpress/data';
+import { canAccessWpcomApis } from 'wpcom-proxy-request';
+import { GeneratorReturnType } from '../mapped-types';
+import { wpcomRequest } from '../wpcom-request-controls';
+import { STORE_KEY } from './constants';
+import type { UpdateLaunchpadNavigatorResponse } from './types';
+import type { APIFetchOptions } from '../queries/use-launchpad-navigator';
+
+export const receiveCurrentChecklist = ( currentChecklist: string ) =>
+	( {
+		type: 'LAUNCHPAD_NAVIGATOR_RECEIVE_CURRENT_CHECKLIST',
+		current_checklist: currentChecklist,
+	} as const );
+
+export function* setCurrentChecklist( siteSlug: string, checklist_slug: string ) {
+	const requestUrl = `/sites/${ siteSlug }/launchpad/navigator`;
```

Comment:
Shouldn't this be just `/launchpad/navigator`? The `/sites/${ siteSlug }` prefix should only be used when we're calling `public-api.wordpress.com`, which won't work on Jetpack sites.
```suggestion
	const requestUrl = `/launchpad/navigator`;
```